index.js

```diff
@@ -8,9 +8,19 @@ var unwatchedTree = require('broccoli-unwatched-tree');
 module.exports = {
   name: 'ember-cli-mirage',
 
-  included: function included(app) {
-    this._super.included.apply(this, arguments);
+  options: {
+    nodeAssets: {
```

Comment:
Hey @dfreeman, I'm wondering where this option `nodeAssets  is been handled. Is it something that Ember CLI does for us? Could you point me in the right direction to learn more about it? Thanks.